Compensation update: Nomad policy and greater options clarity

jamesefhawkins opened the pull request:
Hey Charles

# Nomad policy

@paolodamico mentioned he will be travelling but doesn't know where in advance he'll be going. Our current policy didn't work in that situation, but it'd be good for it to do so.

I've put in a suggestion for how this works, open to feedback

# Options

I put in a little more detail on the main terms for our options policy, since these are positive and should encourage candidates who are looking to understand working here better

piemets commented on contents/handbook/people/compensation.md:
```diff
@@ -56,8 +64,10 @@ If this represents an increase in pay, _we need to approve this change in advanc
 
 For a trip with many destinations over a period of more than 4 months, the location factor will be located along the various places you intend to stay, averaged by the amount of time spent.
+
+If you are uncertain about where you're travelling to in advance, and you are travelling over 4 months, then we will keep your pay the same as when you departed. 4 months later, we will make a manual adjustment to edit your next pay amount based on the average of the previous 4 months. This will take place every 4 months until you are done travelling. If the adjustment would require reclaiming mo ...
```
'...and you are travelling **for** over 4 months'

'...from one of the world's most expensive **areas** to...'

piemets replied:
No comments on the actual approach itself, I think it makes sense.

jamesefhawkins replied:
Fixed

piemets commented on contents/handbook/people/compensation.md:
```diff
@@ -45,6 +45,14 @@ Within each level, we believe there's a place to have incremental steps. We defi
 ​
+Whilst the terms of options for *any company* could vary if we were ever acquired, we have set them up with the following key terms:
+
+* 10 year exercise period
```
I'd say '10 years to exercise your options in the event that you leave Posthog.'

jamesefhawkins replied:
fixed

piemets commented on contents/handbook/people/compensation.md:
```diff
@@ -45,6 +45,14 @@ Within each level, we believe there's a place to have incremental steps. We defi
+Whilst the terms of options for *any company* could vary if we were ever acquired, we have set them up with the following key terms:
+
+* 10 year exercise period
+* Double trigger acceleration which means if we are acquired before you have fully vested, you receive all your options at that time
```
This is not quite correct - I'd say 'Double trigger acceleration, which means that if we are acquired and you are let go or forced to leave due to the company restructuring, you receive all your options at that time.'

(current description is single trigger)

jamesefhawkins replied:
Fixed

jamesefhawkins commented:
@piemets I've addressed all feedback above. Trivial points so no need to double check.

I added a suggestion for how we deal with when we have no data point for someone's location, as we had happen earlier this week - in which case I suggest we base on the cost of living. If no pushback on that, I'll merge this.

jamesefhawkins commented:
Merging for now as above is an easy decision to change later.

corywatilo commented:
👀
